imports/plugins/core/accounts/server/methods/setProfileLanguage.js

```diff
@@ -0,0 +1,57 @@
+import { check, Match } from "meteor/check";
+import R from "ramda";
+import { Accounts, Shops } from "/lib/collections";
+import appEvents from "/imports/node-app/core/util/appEvents";
+import Reaction from "/imports/plugins/core/core/server/Reaction";
+import ReactionError from "@reactioncommerce/reaction-error";
+
+/**
+ * @name accounts/setProfileLanguage
+ * @memberof Accounts/Methods
+ * @method
+ * @param {String} languageCode - i18n language code
+ * @param {String} [accountId] - accountId of user to set language of. Defaults to current user ID
+ * @summary Sets users profile language
+ * @returns {Object} Account document
+ */
+export default function setProfileLanguage(languageCode, accountId) {
```

Comment:
Move this to a non-Meteor mutation function in `/imports/plugins/core/accounts/server/no-meteor/mutations/setProfileLanguage.js` and call it as `const updatedAccount = await context.mutations.setProfileLanguage(context, languageCode, dbAccountId);` in the resolver. Using `callMeteorMethod` is deprecated and soon going away.